examples/disaggregated/slurm/benchmark/start_worker.sh

```
@@ -63,7 +80,14 @@ else
     nsys_prefix="nsys profile -o ${nsys_file} -f true -t cuda,nvtx,python-gil -c cudaProfilerApi --cuda-graph-trace node --capture-range-end=stop --gpu-metrics-devices=none"
 fi
 
+# In-place (.pth-style) TRT-LLM installs may lack the trtllm-serve console
+# script; fall back to the module entry point in that case.
+trtllm_serve_cmd="trtllm-serve"
+if ! command -v trtllm-serve >/dev/null 2>&1; then
+    trtllm_serve_cmd="python3 -m tensorrt_llm.commands.serve"
+fi
+
 ${nsys_prefix} trtllm-llmapi-launch ${numa_bind_cmd} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)